src/main/java/net/sf/jabref/gui/autocompleter/AutoCompleteListener.java

```diff
@@ -410,7 +411,8 @@ private void resetAutoCompletion() {
     }
 
     private String[] findCompletions(String beginning) {
-        return completer.complete(beginning);
+        List<String> results = completer.complete(beginning);
+        return results.toArray(new String[results.size()]);
```

Comment:
Would it make sense to change the method signature to List<String> instead of converting a list into an array?